src/Client/Core/DurableTaskClientOptions.cs

```diff
@@ -13,6 +13,14 @@ public class DurableTaskClientOptions
     DataConverter dataConverter = JsonDataConverter.Default;
     bool enableEntitySupport;
 
+    /// <summary>
+    /// Gets or sets the version of orchestrations that will be created.
+    /// <remarks>
+    /// Currently, this is sourced from the AzureManaged client options.
+    /// </remarks>
```

Comment:
The `<remarks>` tag need to be a peer of the `<summary>` tags. See the other property XML doc comments below as an example.

Reply:
Oops, those must've gotten intertwined by accident. Fixed.